src/content-components/Audio.tsx

```diff
@@ -17,12 +17,21 @@ import {
   RenderRequest,
 } from "./RendererConfig";
 
+
+import { useMediaContext } from "../context/useMediaContext";
+
 type FakeWaveformCanvasProps = {
   audioRef: any;
   uri: string;
 };
 
 const FakeWaveformCanvas = ({ audioRef, uri }: FakeWaveformCanvasProps) => {
+  
+  const { style } = useMediaContext();
```

Comment:
you can now get this as {theme} from the root component props.
`useMediaContext` depends on renderers which this is so using the hook here creates a circular dependency that causes definition issues only sometimes.